suborbital/grav
# Add NATS bridge transport plugin

This PR adds a NATS-based transport to Grav. It also adds a new distinction between normal Transports (i.e. mesh transports) and 'bridge' transports. Bridge Transports are meant to connect to a central broker.

To test, start a NATS server using:
```
docker run -p 4222:4222 -ti nats:latest
```
And then in two different terminals, run:
```
go run transport/nats/tester/main.go
```

They'll each send two messages 5 seconds apart, and you should see them each receive the other.
